src/ResourceManager/Kubernetes/Commands.Kubernetes/Commands/StartAzureRmKubernetesDashboard.cs

```diff
@@ -0,0 +1,330 @@
+// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using System;
+using System.ComponentModel;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Management.Automation;
+using System.Runtime.InteropServices;
+using System.Text;
+using System.Threading;
+using Microsoft.Azure.Commands.Kubernetes.Generated;
+using Microsoft.Azure.Commands.Kubernetes.Models;
+using Microsoft.Azure.Commands.ResourceManager.Common.ArgumentCompleters;
+using Microsoft.Azure.Management.Internal.Resources.Utilities.Models;
+#if NETSTANDARD
+using Microsoft.Extensions.DependencyInjection;
+
+#endif
+
+namespace Microsoft.Azure.Commands.Kubernetes
+{
+    [Cmdlet("Start", KubeNounStr + "Dashboard")]
+    public class StartDashboard : KubeCmdletBase
```

Comment:
OutputType?

Reply:
I'll change it to typeof(KubeTunnelJob).